diff --git a/node_modules/@liquid-carrot/carrot/src/architecture/network.js b/node_modules/@liquid-carrot/carrot/src/architecture/network.js
index 5694f61..e0e2921 100644
--- a/node_modules/@liquid-carrot/carrot/src/architecture/network.js
+++ b/node_modules/@liquid-carrot/carrot/src/architecture/network.js
@@ -456,7 +456,7 @@ function Network(input_size, output_size) {
     }
 
     // Remove gated connections gated by this node
-    for (i = node.gated.length - 1; i >= 0; i--) {
+    for (let i = node.gated.length - 1; i >= 0; i--) {
       const connection = node.gated[i];
       self.ungate(connection);
     }
diff --git a/node_modules/@liquid-carrot/carrot/src/neat.js b/node_modules/@liquid-carrot/carrot/src/neat.js
index a1c7c93..818045b 100644
--- a/node_modules/@liquid-carrot/carrot/src/neat.js
+++ b/node_modules/@liquid-carrot/carrot/src/neat.js
@@ -176,7 +176,7 @@ const Neat = function(inputs, outputs, dataset, options) {
     size = size || self.population_size
 
     // Prioritize network, otherwise use template network, otherwise use "new network"
-    copyNetwork = network
+    const copyNetwork = network
       ? () => network.clone()
       : self.template
       ? () => self.template.clone()
